python/ray/experimental/data/dataset.py

```diff
@@ -359,46 +363,124 @@ def write_parquet(path: str,
         Time complexity: O(dataset size / parallelism)
 
         Args:
-            path: The path in the filesystem to write to.
+            path: The path to the destination root directory, where Parquet
+                files will be written to..
             filesystem: The filesystem implementation to write to.
         """
-        raise NotImplementedError  # P0
+        import pyarrow.parquet as pq
+
+        @ray.remote
+        def parquet_write(write_path, block):
+            print(f"Writing {block.num_rows()} records to {write_path}.")
```

Comment:
Can we switch to logger.debug()?

Reply:
Sure, I was matching existing code. I'll port existing print statements as well.